docs/book/reference/environment-variables.md

```diff
@@ -129,6 +129,28 @@ To set the path to the global config file, used by ZenML to manage and store the
 export ZENML_CONFIG_PATH=/path/to/somewhere
 ```
 
+## CLI output formatting
+
+### Default output format
+
+Set the default output format for all CLI list commands:
+
+```bash
+export ZENML_DEFAULT_OUTPUT=json
```

Comment:
Just a very small inconsistency that I found while testing this: 

<img width="782" height="523" alt="Image" src="https://github.com/user-attachments/assets/d13a8984-de71-493b-89dd-666983006350" />

The columns names in the regular list are all upper case, while the json outputs have half-upper half-lower case. Should be easy to fix. If it is not, feel free to ignore for now.